ProteoformSuiteInternal/Lollipop.cs

```diff
@@ -316,9 +338,9 @@ private static string GetOneGiantProtein(IEnumerable<Protein> proteins, bool met
         }
 
         //ET,ED,EE,EF COMPARISONS
-        public static double ee_max_mass_difference = 250; //TODO: implement this in ProteoformFamilies and elsewhere
-        public static double et_low_mass_difference = -100;
-        public static double et_high_mass_difference = 200;
+        public static double ee_max_mass_difference = 500; //TODO: implement this in ProteoformFamilies and elsewhere
```

Comment:
I changed it to 500 just for testing purposes to compare with the old way... I think even for labeled it's unnecessary so I'll change it back!